backend/attendance-service/src/main/java/org/egov/repository/AttendanceRepository.java

```diff
@@ -20,48 +20,50 @@
 public class AttendanceRepository {
```

Comment:
This would cause confusion. Rename to `AttendanceRegisterRepository` or `AttendanceLogRepository`.